dev/io.openliberty.org.apache.myfaces.4.0_fat/fat/src/io/openliberty/org/apache/myfaces40/fat/tests/AnnotationLiteralsTest.java

```diff
@@ -37,6 +38,7 @@
  * Tests to ensure new Annotation Literals can be used in an expected way.
  */
 @RunWith(FATRunner.class)
+@SkipForRepeat(SkipForRepeat.EE11_FEATURES) // Errors in EE11
```

Comment:
Do we have an issue opened to go back and address this?

Reply:
Yes - https://github.com/OpenLiberty/open-liberty/issues/27628